gcp_variant_transforms/beam_io/vcf_parser.py

```diff
@@ -41,7 +46,7 @@
 DEFAULT_PHASESET_VALUE = '*'  # Default phaseset value if call is phased, but
                               # no 'PS' is present.
 MISSING_GENOTYPE_VALUE = -1  # Genotype to use when '.' is used in GT field.
-
+FIRST_HEADER_LINE = '##fileformat=VCFv4.2\n'
```

Comment:
I think we should use 4.0 as the 'least common denominator'. Not sure if it matters for the parser though.